packages/testing/src/test/dispose.unit.ts

```diff
@@ -6,9 +6,9 @@ describe('dispose', () => {
         const events = [] as string[];
 
         it('setup dispose after', () => {
-            disposeAfter(() => events.push('first in default group'));
-            disposeAfter(() => events.push('second in default group'));
-            disposeAfter(() => events.push('last in default group'));
+            disposeAfter(() => events.push('first in default group'), 'first default');
+            disposeAfter(() => events.push('second in default group'), 'second default');
+            disposeAfter(() => events.push('last in default group'), 'last default');
```

Comment:
I already commented in the `disposeAfter` definition - it would be nice to have the same api signature as `disposables.add`

Reply:
this function going to be deleted. so I don't want to change it and do all the work to fix it and then delete.